src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculable, Printable {
+    protected Color color;

```

Comment:
Don't begin class or method implementation with an empty line. The empty line after the class declaration should be removed to follow best practices.